psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -1227,18 +1194,20 @@ void HF::check_phases() {
 }
 
 void HF::print_occupation() {
-    std::vector<std::string> labels = molecule_->irrep_labels();
-    std::string reference = options_.get_str("REFERENCE");
+    auto labels = molecule_->irrep_labels();
+    auto reference = options_.get_str("REFERENCE");
     outfile->Printf("          ");
     for (int h = 0; h < nirrep_; ++h) outfile->Printf(" %4s ", labels[h].c_str());
     outfile->Printf("\n");
+    auto docc = doccpi();
+    auto socc = soccpi();
     outfile->Printf("    DOCC [ ");
-    for (int h = 0; h < nirrep_ - 1; ++h) outfile->Printf(" %4d,", doccpi_[h]);
-    outfile->Printf(" %4d ]\n", doccpi_[nirrep_ - 1]);
+    for (int h = 0; h < nirrep_ - 1; ++h) outfile->Printf(" %4d,", docc[h]);
+    outfile->Printf(" %4d ]\n", docc[nirrep_ - 1]);
     if (reference != "RHF" && reference != "RKS") {
         outfile->Printf("    SOCC [ ");
-        for (int h = 0; h < nirrep_ - 1; ++h) outfile->Printf(" %4d,", soccpi_[h]);
-        outfile->Printf(" %4d ]\n", soccpi_[nirrep_ - 1]);
+        for (int h = 0; h < nirrep_ - 1; ++h) outfile->Printf(" %4d,", socc[h]);
+        outfile->Printf(" %4d ]\n", socc[nirrep_ - 1]);
```

Comment:
The code is not printing out AOCC and BOCC

Reply:
Correct. The _old behavior_ was to print out DOCC and SOCC. I didn't want to change it to print out AOCC and BOCC - too much for one PR.

Reply:
Does the code already print out the number of alpha and beta electrons? I don't think it does.

Since the code changes the way the calculation is run, the printout should be changed as well. Just add printouts for AOCC and BOCC?

Reply:
AOCC and BOCC are printed in the "excited MOM" case. I'm happy to change to print alpha and beta always, print docc and socc if well-defined. Again, fine with @loriab ?

Reply:
more printing? sure!

Reply:
Changed proposal to "print alpha and beta always, print docc and socc **always**." A warning will fire if those might be misleading, and AOCC and BOCC should clear confusion.